dbt_project.yml

```diff
@@ -104,9 +104,6 @@ models:
       polygon:
         +schema: nft_polygon
         +materialized: view
-      optimism:
```

Comment:
Is this removal desired?

Reply:
oh, no that must have been some merge bug, fixing it.